packages/ledger-icrc/src/index.canister.ts

```diff
@@ -30,12 +31,19 @@ export class IcrcIndexCanister extends Canister<IcrcIndexService> {
    * because the index canisters makes a call to the ledger canister to get the transaction data.
    * Index Canister only holds the transactions ids in state, not the whole transaction data.
    */
-  getTransactions = async (
-    params: GetAccountTransactionsParams,
-  ): Promise<GetTransactions> => {
+  /**
+   * Get the transactions of an account.
+   *
+   * @param {GetAccountTransactionsParams} params The parameters to get the transactions of an account.
+   * @returns {Promise<GetTransactions>} The list of transactions and further related information of the given account.
+   */
+  getTransactions = async ({
+    certified,
```

Comment:
This is a breaking change, right?

Reply:
No because `certified` is an additional optional parameter.

```
export type GetAccountTransactionsParams = {
  max_results: bigint;
  start?: IcrcTxId;
  account: IcrcAccount;
} & QueryParams;

export interface QueryParams {
  certified?: boolean;
}
```

Reply:
But before the default was doing an `update` call. Now it will do a query call. It's not the same.

Are all the old SNSes using the new index canister? What would happen if the NNS dapp makes a query call to an old index canister? That won't work, right?

Reply:
The default is `update`.

```
protected caller = ({ certified = true }: QueryParams): T =>
    certified ? this.certifiedService : this.service;
```

> Are all the old SNSes using the new index canister?

No. It's up to the SNSes to upgrade. The index-ng is available for upgrade in the SNS-W canister. Dragginz has upgraded already.

> What would happen if the NNS dapp makes a query call to an old index canister? That won't work, right?

Indeed that won't work but, NNS dapp does not make query call to the ICRC index canister yet.

Moreover, you cannot begin to implement query call to an index canister in NNS dapp or in any dapps if ic-js does not expose the function.

Reply:
Thanks! I forgot the caller had `true` as default.

> Indeed that won't work but, NNS dapp does not make query call to the ICRC index canister yet.

I meant it because I thought the default after this PR would be `query`. Now it's clear.